crates/control/src/motion/motor_commands_optimizer.rs

```diff
@@ -29,6 +29,7 @@ impl MotorCommandsOptimizer {
         let mut motor_commands = *context.motor_commands;
         motor_commands.stiffnesses.left_arm.hand = 0.0;
         motor_commands.stiffnesses.right_arm.hand = 0.0;
```

Comment:
Do we actually still need this node? This is not really optimizing anything?! Maybe just move this zero stiffness for hands to the command sender and drop the entire node?

Reply:
Hmm I think It's better if this is done in it's own node, unless you think the overhead is too large for the readability benefit.

Reply:
It is a lot of overhead to only set 2 floats to zero.
My initial intention actually was to remove the optimization part. At least the node's name does not fit anymore. It is not optimizing anything

Reply:
Okay, then let's move this to the `command_collector`. Issue #708